src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+/* #region card */
+$cardWidth: 200px;
```

Comment:
You have duplicated variables in 3 different places - here, in file `variables.scss` and in file `stars.scss`. Please, clean up your code.